### PR TITLE
Add CVE-2016-3098 for administrate gem

### PR DESCRIPTION
**Pull request by tute**

### CODE REVIEW

File: gems/administrate/CVE-2016-3098.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+gem: administrate
+cve: 2016-3098
+title: Cross-site request forgery (CSRF) vulnerability in administrate gem
+date: 2016-04-01
+
```
Comment (reedloden) [inline gems/administrate/CVE-2016-3098.yml]: Can you add a `url` of something like http://seclists.org/oss-sec/2016/q2/0 or some other public advisory? `url` is a required option.

File: gems/administrate/CVE-2016-3098.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+description: >-
+  `Administrate::ApplicationController` actions didn't have CSRF
+  protection. Remote attackers can hijack user's sessions and use any
+  functionality that administrate exposes on their behalf.
+
+unaffected_versions:
```
Comment (reedloden) [inline gems/administrate/CVE-2016-3098.yml]: Not needed here, as `patched_versions` covers this. Just remove `unaffected_versions` part.

**Comment by tute** [comment]
Added a URL, but I don't think it's published yet. Thanks!

File: gems/administrate/CVE-2016-3098.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+gem: administrate
+cve: 2016-3098
+title: Cross-site request forgery (CSRF) vulnerability in administrate gem
+date: 2016-04-01
+url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=2016-3098
```
Comment (reedloden) [inline gems/administrate/CVE-2016-3098.yml]: Ah, looking for more of an advisory here. Why not http://seclists.org/oss-sec/2016/q2/0 or something like that?

**Comment by tute** [comment]
> Ah, looking for more of an advisory here. Why not http://seclists.org/oss-sec/2016/q2/0 or something like that?

Right, fixed. Thanks!

**Comment by reedloden** [comment]
Thanks!

**Pull request merged**